staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -89,6 +89,11 @@ type KubeSchedulerConfiguration struct {
 	// with the extender. These extenders are shared by all scheduler profiles.
 	// +listType=set
 	Extenders []Extender `json:"extenders,omitempty"`
+
+	// MaintainCacheWhenNonLeader enables caching for non-leader candidates. This makes non-leader candidates
+	// start with watching cache. Doing so will have faster failover at cost of higher memory overhead.
+	// Defaults to true.
+	MaintainCacheWhenNonLeader *bool `json:"maintainCacheWhenNonLeader,omitempty"`
```

Comment:
it would be nice if we could name this such that a default of false matches the current behavior, and avoid the need for fixup in non-v1 conversion files

possible suggestion:

`DelayCacheUntilActive bool`

that name also makes more sense in a scheduler that doesn't use leader election (it becomes active immediately)

Reply:
Changed as suggested, thanks!